File: src/scanners/PPSSPPFlatpakSource.js
const PPSSPPSource = require("./PPSSPPSource.js");
const PPSSPPFlatpakGame = require("../games/PPSSPPFlatpakGame.js");

const USER_DIR = process.env["HOME"];

class PPSSPPFlatpakSource extends PPSSPPSource{

	static name = "PPSSPP (Flatpak)";
	static gameClass = PPSSPPFlatpakGame;

	configPath = `${USER_DIR}/.var/app/org.ppsspp.PPSSPP/config/ppsspp/PSP/SYSTEM/ppsspp.ini`;

}

module.exports = PPSSPPFlatpakSource;

File: gali/src/sources/ppsspp/flatpak/ppsspp_flatpak_source.py
from gali.sources.ppsspp.flatpak.ppsspp_flatpak_game import PPSSPPFlatpakGame
from gali.sources.ppsspp.ppsspp_source import PPSSPPSource
from gali.utils.locations import HOME


class PPSSPPFlatpakSource(PPSSPPSource):

    name: str = "PPSSPP (Flatpak)"
    game_class: type[PPSSPPFlatpakGame] = PPSSPPFlatpakGame
    config_path: str = f"{HOME}/.var/app/org.ppsspp.PPSSPP/config/ppsspp/PSP/SYSTEM/ppsspp.ini"